managing_providers/_topics/embedded_workflows.md

```diff
@@ -280,7 +302,6 @@ Workflows must be authored in Amazon State Languages (ASL) format. As part of au
   }
 }
 ```
-
```

Comment:
Was this removal intentional?  It doesn't seem related to the rest of these changes